bindings/cs/rl.net/rl.net.csproj

```diff
@@ -26,4 +28,37 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
-</Project>
+  <ItemGroup>
+    <PackageReference Include="Clarius.TransformOnBuild" Version="1.22.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="InternalsVisibleToTest.cs">
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="InternalsVisibleToTest.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>InternalsVisibleToTest.cs</LastGenOutput>
+    </None>
+    <Compile Include="InternalsVisibleToTest.cs" Exclude="@(Compile)">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>InternalsVisibleToTest.tt</DependentUpon>
+    </Compile>
+  </ItemGroup>
+
+  <Target Name="OutputVars" BeforeTargets="Build">
```

Comment:
Are these output vars still needed?

Reply:
Still have a number of steps to finish.  I'd like to merge this to release with this on.  Can get rid of if eventually.